src/Telemetry/AzureDevOpsClient/AzureDevOpsHttpClientFactory.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Internal.AzureDevOps
+{
+    public class AzureDevOpsHttpClientFactory : IHttpClientFactory
```

Comment:
I think in this situation this is the easier solution because AzureDevOpsClient is being created via AzureDevOpsClientFactory by some other services. If we use this this approach, it's very easy not to change the behavior of those other services, and not to have to fix tests, etc..

Reply:
@ChadNedzlek I removed the factory and used `HttpMessageHandlerBuilder` to configure the delegating handler. I also modified the other app (`DotNet.Status.Web` ) to use the `RetryAfterHandler`. I *think* that it doesn't use ServiceFabric so I added null checking to the `RetryAfterHandler` `TelemetryClient`

Reply:
@Chrisboh - as discussed on Tue, since @ChadNedzlek is OOF, could someone from your team (perhaps @MattGal or @alexperovich) follow up on this and unblock @dkurepa? Thanks!